Makefile-az.mk

```diff
@@ -358,3 +358,11 @@ az-helm-install-snapshot: az-configure-values ## Install Karpenter snapshot rele
 
 az-rmcrds: ## Delete Karpenter CRDs
 	kubectl delete crd nodepools.karpenter.sh nodeclaims.karpenter.sh aksnodeclasses.karpenter.azure.com
+
+az-swagger-generate-clients-raw:
+	cd pkg/provisionclients && swagger generate client -f swagger/*.json
```

Comment:
A couple of thoughts:
* You should be able to remove make targets / fold them into just "go generate" by creating e.g. provisionclients/generate.go with something like `//go:generate swagger generate client ...` etc.
* Take a look at generate client options, consider maybe `--keep-spec-order`?
* Could we maybe keep the spec in yaml instead of json?

Reply:
(will circle back to this later, along with the rest of the suggestions)